src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -80,14 +82,10 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
 
     public static function castPeriod(\DatePeriod $p, array $a, Stub $stub, $isNested, $filter)
     {
-        if (defined('HHVM_VERSION_ID') || \PHP_VERSION_ID < 50620 || (\PHP_VERSION_ID >= 70000 && \PHP_VERSION_ID < 70005)) { // see https://bugs.php.net/bug.php?id=71635
-            return $a;
-        }
-
         $dates = array();
         if (\PHP_VERSION_ID >= 70107) { // see https://bugs.php.net/bug.php?id=74639
```

Comment:
Minimal current PHP version is 7.1.3. What about bump it to 7.1.7 to avoid this condition?

Reply:
a caster is not enough a reason to bump the minimum version